src/main/scala/zio/ftp/package.scala

```diff
@@ -0,0 +1,15 @@
+package zio
+
+import java.io.IOException
+
+import zio.blocking.Blocking
+import zio.macros.delegate.Mix
+
+package object ftp {
+
+  def taskIO[A](f: => A): ZIO[Any, IOException, A] = Task(f).refineOrDie {
```

Comment:
This one should probably use `zio.blocking.blocking { ... }`. Maybe even `zio.blocking.effectBlocking { ... }`.